scripts/run-percy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# skip percy builds when this is not a pull request or when it's not master
+if [[ $TRAVIS_PULL_REQUEST != 'false' || ${TRAVIS_BRANCH} == 'master' ]]
+  then
+  # skip percy builds when pull request is opened by a robot
+  regexp='^renovate\/'
+  if ! [[ ${TRAVIS_PULL_REQUEST_BRANCH} =~ ${regexp} ]]
+    then
+      yarn percy
+    else
+      echo 'Skipping percy - PR opened by `renovate`'
```

Comment:
Maybe add a note why we do this. It seems intuitive to run percy when deps update.

Reply:
But we already run percy on `master`, so if someone were to merge one of these PRs, it will run.

Reply:
comment updated [here](https://github.com/commercetools/ui-kit/pull/302/commits/c52f18d368154613f20df0988c942c12ba9aeb56)

Reply:
Thanks.